pkg/storage/snapshot/snapshot_base.go

```diff
@@ -74,6 +74,7 @@ type VMSnapshotController struct {
 	VMInformer                cache.SharedIndexInformer
 	VMIInformer               cache.SharedIndexInformer
 	StorageClassInformer      cache.SharedIndexInformer
+	StorageProfileInformer    cache.SharedIndexInformer
```

Comment:
I've seen this effort recently to use stores when the actual informers are not needed:
https://github.com/kubevirt/kubevirt/pull/12126/commits/0e327730d95c79faa128d6d49e6b9e8c03977c9f
And more recently reference
https://github.com/kubevirt/kubevirt/commit/a657a7eb5a2e27f4d786090e6e2e937291389ea4

Reply:
If you dont mind Ill keep it like this for this PR and fix all possible informers-> store in a followup that will not need to be backported

Reply:
sure